docs/Farming & Staking/Staking/operators.mdx

```diff
@@ -64,52 +65,52 @@ Ensure you replace `your_domain_id` with your domain identifier in the command.
 <Tabs groupId="OS">
 <TabItem value="windows" label="🖼️ Windows" default>
 
-```powershell
-target/production/subspace-node `
---chain gemini-3g `
---bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp `
---name your_node_name `
---base-path your_path_to_node_data `
--- `
---domain-id your_domain_id `
---chain gemini-3g `
---operator `
---keystore-path /tmp/keystore 
-```
+  ```powershell
+  target/production/subspace-node `
+  --chain gemini-3g `
+  --bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp `
```

Comment:
`bootnodes` should be after the `--`.

> `bootnodes` for domains should into the part of arguments that corresponds to domains, which is after `--`. Otherwise you override consensus bootstrap address, which will result in both consensus and domain side being offline.

Reply:
Note that the screenshot just below this section will be incorrect after fixing this. As the image is to show what a successfully started node looks like, maybe we should chop the command out as it's another thing to keep in sync and has caused confusion.

Reply:
That's a really good idea Jim, will do!